packages/runtime/package.json

```
@@ -24,17 +24,27 @@
         "cdep": "tsc && madge --circular dist",
         "lint:tsc": "tsc --noEmit && tsc -p test/tsconfig.json --noEmit",
         "test:ci:lokijs": "USE_LOKIJS=true jest -i",
-        "test:ci:mongodb": "jest -i",
+        "test:ci:mongodb": "jest -i  --coverage=true",
```

```suggestion
        "test:ci:mongodb": "jest -i --coverage=true",
```

same for all other places.